src/components/header.js

```diff
@@ -0,0 +1,142 @@
+import { useState } from 'react';
+import { Chromicons } from './icons/chromicons';
+import { CheckCircle, Download, Flag, Lifeology } from '@lifeomic/chromicons';
+import * as allLinedChromicons from '@lifeomic/chromicons';
+import Head from 'next/head';
+import clsx from 'clsx';
+
+const HeroNavLink = ({ className, children, ...rootProps }) => {
+  return (
+    <a
+      className={clsx(
+        'flex items-center px-2 duration-300 ease-in-out transition-opacity hover:opacity-75 focus:outline-none focus-visible:shadow-dark-mode-outline focus-visible:underline',
+        className
+      )}
+      {...rootProps}
+    >
+      {children}
+    </a>
+  );
+};
+
+export const Header = ({ pkgVersion }) => {
+  return (
+    <>
+      <Head>
+        <title>CHROMICONS</title>
+        <link
+          rel="shortcut icon"
+          href={`${process.env.BACKEND_URL}/favicon.ico`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href={`${process.env.BACKEND_URL}/favicon-16x16.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href={`${process.env.BACKEND_URL}/favicon-32x32.png`}
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href={`${process.env.BACKEND_URL}/apple-touch-icon.png`}
+        />
+        <link
+          rel="apple-touch-icon-precomposed"
+          type="image/png"
+          sizes="144x144"
+          href={`${process.env.BACKEND_URL}/chroma@2x.png`}
+        />
+        <link
+          rel="apple-touch-icon-precomposed"
+          type="image/png"
+          sizes="114x114"
+          href={`${process.env.BACKEND_URL}/chroma@2x.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="192x192"
+          href={`${process.env.BACKEND_URL}/android-icon-192x192.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="512x512"
+          href={`${process.env.BACKEND_URL}/android-icon-512x512.png`}
+        />
+        <link
+          rel="manifest"
+          href={`${process.env.BACKEND_URL}/manifest.json`}
+        />
+        <meta name="msapplication-TileColor" content="#ffffff" />
+        <meta
+          name="msapplication-TileImage"
+          content={`${process.env.BACKEND_URL}/chroma@2x.png`}
+        />
+        <meta name="theme-color" content="#ffffff" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0"
+        ></meta>
+      </Head>
+      <header>
+        <div className="flex justify-between mb-16 w-3/4 m-auto mx-auto">
+          <div className="py-12 flex items-center text-white -space-x-8">
+            <div style={{ cursor: 'pointer' }}>
+              <Chromicons />
+            </div>
```

Comment:
It looks like `<Chromicons` is a link, does the cursor pointer not get added there by default?  I don't think we want the pointer on the surrounding div, but rather the actual link.  May want to remove this wrapping `<div` and see if the `<Chromicons` component has the cursor pointer set by default.

Reply:
Doesn't default to cursor pointer on the link, added it to the svg.

Reply:
I think we still want to use Link in this case since it's not an external link

https://nextjs.org/docs/api-reference/next/link